src/content-scripts/clientAPI.js

```diff
@@ -425,6 +435,29 @@ function importPublicKey(keyringId, armored, callback) {
   });
 }
 
+function lookupPublicKey(keyringId, emailAddr, source, callback) {
```

Comment:
use an object instead of emailAdrr and source so that we can easily extend this for lookup by fingerprint or other options